tower_cli/resources/workflow.py

```diff
@@ -45,16 +45,19 @@ def __init__(self, data, wfjt, include_id=False):
             else:
                 node_attrs[fd] = data[fd]
         node_attrs['workflow_job_template'] = wfjt
+        ujt_ambiguity_msg = (
+            'You should provide exactly one of the attributes'
+            ' job_template, project, workflow, inventory_source, or unified_job_template.'
```

Comment:
:+1:  for this clear message. Although I am not familiar with `unified_job_template`